glib/src/variant_dict.rs

```diff
@@ -130,12 +130,12 @@ impl VariantDict {
     /// instance already, you should use the [`insert_value()`](#method.insert_value)
     /// method instead.
     #[doc(alias = "g_variant_dict_insert_value")]
-    pub fn insert<T: ToVariant>(&self, key: &str, value: &T) {
+    pub fn insert<T: Into<Variant>>(&self, key: &str, value: T) {
```

Comment:
This could also be `impl Into<Variant>`. Less noise

Reply:
Also this is the only place where this is used currently. Do you have specific code where you want to make use of this / where this makes things easier?

Reply:
I have used `Settings::set`, but also I will want to use this in the class macro for gio actions and widget actions that have return values.

Reply:
Sounds good